pkgs/tools/security/sequoia-chameleon-gnupg/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, pkg-config
+, nettle
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sequoia-chameleon-gnupg";
+  version = "0.1.0";
+
+  src = fetchFromGitLab {
+    owner = "sequoia-pgp";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-msJOQUqq0S+cttSucvXxAFIcFu3R4lUCgIpkaumkaZY=";
+  };
+
+  cargoHash = "sha256-rnZPZY4mg18Plwcrppn7a1+H6CI/A4ntQ5TsZvi6kcA=";
+
+  nativeBuildInputs = [ rustPlatform.bindgenHook pkg-config ];
+  buildInputs = [ nettle ];
+
+  # gpgconf: error creating socket directory
+  doCheck = false;
```

Comment:
I just can't make the tests work, to reproduce the issue:
```nix
runCommand "test" {
  nativeBuildInputs = [ gnupg ];
} "HOME=$TMPDIR gpgconf --create-socketdir"
```

Reply:
I think sockets don't work in Nix' sandbox, it's not so bad to disable tests due to this. We can still maintain this package.

Reply:
@NickCao would you like to keep things as are regarding the tests?

Reply:
I don't have the time to find a fix these days, I'd leave it for later.

Reply:
I think we can merge this as is in the meantime.